src/app/feedback/feedback.component.ts

```diff
@@ -70,7 +70,7 @@ export class FeedbackComponent implements OnInit, AfterViewInit, OnDestroy {
     this.user = this.userService.get();
     this.getFeedback();
     this.feedback.filterPredicate = composeFilterFunctions([ filterDropdowns(this.filter), filterSpecificFields([ 'owner', 'title' ]) ]);
-    this.feedback.sortingDataAccessor = (item, property) => item[property].toLowerCase();
+    this.feedback.sortingDataAccessor = (item, property) => item[property].toString().toLowerCase();
```

Comment:
item[property] return date which is long int so it don't have toLowerCase() method. just remove this method.
this.feedback.sortingDataAccessor = (item, property) => item[property];

Reply:
See the CommunityComponent `sortingDataAccessor` function.  That checks the type of `item[property]` and applies the `toLowerCase` method if it is a string.

Rather than duplicate, you can add that function to `src/app/shared/table-helpers.ts` and use in both places.